pattern-library/sass/patterns/_layouts.scss

```diff
@@ -3,97 +3,29 @@
 
 // About: Contains base settings and options for the preset default Pattern Library UI Layouts.
 
-// #SETTINGS
-// #GLOBAL
-// #UTILITY
 // #CANNED - IN VIEW
 // #CANNED - VIEW
 
-
-// ----------------------------
-// #SETTINGS
-// ----------------------------
 @import 'grid';
 
-$susy-flush:            layout($susy-default inside); // set susy to make columns flush
-$susy-reversed:         layout($susy-default $grid-direction-reversed); // set susy to make columns display in reverse order
-$susy-view:             layout($susy-default auto); // set susy to make columns view display in reverse order
-$susy-view-flush:       layout($susy-default auto inside); // set susy to make view columns display in reverse order
-$susy-view-reversed:    layout($susy-default auto $grid-direction-reversed); // set susy to make view columns display in reverse order
-
-
-// ----------------------------
-// #GLOBAL
-// ----------------------------
-
-
-// ----------------------------
-// #UTILITY
-// ----------------------------
-// CASE: Layout with gutter-separated columns
-%layout-default {
-    @include container($susy-default);
-}
-
-// CASE: Layout with flush columns
-%layout-flush {
-    @include container($susy-flush);
-}
-
-// CASE: Layout with reversed columns
-%layout-reversed {
-    @include container($susy-reversed);
-}
-
-// CASE: View-based layout
-%layout-view {
-    @include container($susy-view);
-}
-
-// CASE: View-based layout with flush columns
-%layout-view-flush {
-    @include container($susy-view-flush);
-}
-
-// CASE: View-based layout with reversed columns
-%layout-view-reversed {
-    @include container($susy-view-reversed);
-}
-
 // basic layout column
-%layout-column {
-    @include span(full); // plan for mobile first
+@mixin layout-column {
+    @include span(12); // plan for mobile first
 }
 
-
 // ----------------------------
 // #CANNED - IN VIEW
 // ----------------------------
 
-// basic layout
-.layout {
-    @extend %layout-default;
-}
-
-// basic layout - flush
-.layout-flush {
-    @extend %layout-flush;
-}
-
-// basic columns
-.layout-col {
-    @extend %layout-column;
-}
-
 // Full Layout
 .layout-full {
-    @extend %layout-default;
+    @include grid-container;
 
     .layout-col-a,
     .layout-col-b,
     .layout-col-c,
     .layout-col-d {
-        @include span(full);
+        @include span(12);
```

Comment:
So I thought about this a bit, and I think anywhere where we're providing classes you can consume directly we should stick with 12 as the basis (and thus hardcode fractions of 12) because it's assumed that if you're using those classes, you don't want to do any customization.

However, in places where we're providing mixins or extendable classes for you to use to build your own layout system, we should use variables, because we want to support overriding the default number of columns for those cases.